proposals/lip-0046.md

```diff
@@ -54,7 +54,7 @@ We define the following constants:
 | `ADDRESS_LENGTH`               | uint32   | 20              | Length in bytes of type Address.                |
 | `SEED_LENGTH`                  | uint32   | 16              | Length in bytes of a valid seed revealed.       |
 | **Config parameters**          |          | Mainchain value |                                                 |
-| `MAX_LENGTH_VALIDATOR_REVEALS` | uint32   | 206             | Maximum length of the `validatorReveals` array. |
+| `MAX_LENGTH_VALIDATOR_REVEALS` | uint32   | 206             | Maximum length of the `validatorReveals` array. It should be set as twice the maximum length of the chains validator set. |
```

Comment:
That's a good point. I think it should be config parameter since it somehow depends on other module's value. It should also be compatible with PoA or whatever consensus module SDK provides (this is why it is written in the form "the maximum length of the chains validator set" and not in PoS-specific language). 

Also I don't think it is necessary to be 2 * length, it's just a recommendation (this we need to ask to make sure it's true). We need it to be 2 *length for the commit-and-reveal process but in general it could have other values. Let's leave it open for now and discuss it again.